styleguide/derek/templates/index.jade

```diff
@@ -87,3 +85,10 @@
       a.red(href="derek/templates/landing-page/landingpage-tradeshow-event") Tradeshow Event Landing Page
     li
       a.red(href="derek/templates/landing-page/landingpage-webinar") Webinar Landing Page
+    li
+      a.red(href="derek/templates/landing-page/landingpage-thankyou1") Thank You v1
+    li
+      a.red(href="derek/templates/landing-page/landingpage-thankyou2") Thank You v2
+    li
+      a.red(href="derek/templates/landing-page/landingpage-thankyou3") Thank You v3
+
```

Comment:
You must have missed my comment yesterday about not needing to actually produce all three variants here. Let's just leave the 2 column version and drop the other two, otherwise it's too noisy and if we have to make a change to one, we'll have to make a change to all three.  

We already have coverage in other ways to see the related resource variants anyway, so we don't need it here.